newspack-theme/js/src/post-subtitle/utils.js

```diff
@@ -12,7 +12,11 @@ export const META_FIELD_NAME = 'newspack_post_subtitle';
  * @param  {string} subtitle Subtitle text
  */
 export const appendSubtitleToTitleDOMElement = ( subtitle, isInCodeEditor ) => {
-	const titleEl = document.querySelector( '.editor-post-title__block' );
+	let titleEl = document.querySelector( '.editor-post-title__block' ); // Legacy selector
```

Comment:
FYI, you can include multiple selectors in one `querySelector` call and it'll return the first matching element. e.g.:

```suggestion
	const titleEl = document.querySelector(
		'.editor-post-title__block, .edit-post-visual-editor__post-title-wrapper'
	);
```

Reply:
Thanks @dkoo! 

I think I'm going to leave this as is, just so it's (hopefully!!) clearer to me later which one to "retire" after 5.9 has been around for a while :)